pkgs/top-level/all-packages.nix

```diff
@@ -12642,6 +12644,7 @@ julia_15 = callPackage ../development/compilers/julia/1.5.nix {
     libmfx = if stdenv.isDarwin then null else intel-media-sdk;
     libpulseaudio = if stdenv.isDarwin then null else libpulseaudio;
     rav1e = if stdenv.isDarwin then null else rav1e;
+    svt-av1 = if stdenv.isDarwin then null else svt-av1;
```

Comment:
This line is not necessary, it builds on darwin. Would you mind removing it?
Besides that, LGTM